bezier-rs/lib/src/lib.rs

```diff
@@ -70,18 +70,61 @@ impl Bezier {
 		}
 	}
 
+	// TODO: Maybe move to a utils file
+	fn compute_abc(start_point: DVec2, point_on_curve: DVec2, end_point: DVec2, t_term: f64, one_minus_t_term: f64) -> [DVec2; 3] {
```

Comment:
I don't love the names t_term or one_minus_t_term. t_to_nth_power is more clear but also kinda gross. t_power might be a good middle ground.